Kitodo/src/test/java/org/kitodo/MockDatabase.java

```diff
@@ -291,28 +294,27 @@ public static void insertLdapGroups() throws DAOException {
     }
 
     private static void insertProcesses() throws DAOException, DataException {
-        Project project = serviceManager.getProjectService().getById(1);
-
         Process firstProcess = new Process();
         firstProcess.setTitle("First process");
         firstProcess.setOutputName("Test");
         firstProcess.setWikiField("wiki");
         LocalDate localDate = new LocalDate(2016, 10, 20);
         firstProcess.setCreationDate(localDate.toDate());
         firstProcess.setSortHelperImages(20);
-        List<Batch> batches = new ArrayList<>();
         Batch firstBatch = serviceManager.getBatchService().getById(1);
         Batch secondBatch = serviceManager.getBatchService().getById(3);
         List<Process> processes = new ArrayList<>();
         processes.add(firstProcess);
         firstBatch.setProcesses(processes);
         secondBatch.setProcesses(processes);
+        List<Batch> batches = new ArrayList<>();
         batches.add(firstBatch);
         batches.add(secondBatch);
         firstProcess.setBatches(batches);
         firstProcess.setTemplate(true);
         firstProcess.setInChoiceListShown(true);
         firstProcess.setDocket(serviceManager.getDocketService().getById(1));
+        Project project = serviceManager.getProjectService().getById(1);
```

Comment:
Changing the ordering is also not necessary.

Reply:
It is required because the distance between variable assignment and use must not be greater than three lines.